pkg/kubelet/container_bridge.go

```diff
@@ -82,6 +82,16 @@ func ensureCbr0(wantCIDR *net.IPNet) error {
 		}
 		return createCBR0(wantCIDR)
 	}
+	// Put the container bridge into promiscuous mode to force it to accept hairpin packets.
```

Comment:
yeah.. lets do it the first time around too so that the intent is captured that we want it right away too..